astropy/units/format/unicode_format.py

```diff
@@ -5,7 +5,9 @@
 """
 
 
-from . import console
+import astropy.units as u
+
+from . import console, utils
```

Comment:
Is there any benefit to use local import over absolute import nowadays?

Reply:
I think following precedent in `latex` is fine. I personally like that it is immediately clear that it is a related module (but am very happy we got rid of using `..`!)

Reply:
There are relative imports in a bunch of places, and I didn't want to touch any linting choices here 😅 
Does it make sense to send a new PR which fixes relative imports all across astropy?

Reply:
I think you made the right choice, certainly for this PR. For changing imports across astropy, I think there are better things to do with both your and reviewer time - I think we are in a decent state and to be honest I'm getting a bit tired of the churn caused by trying to standardize on what sometimes seems to be just the latest fashion. More practically, at some point `units` should really be split of into a standalone package, and then having those relative imports means less work...